drivers/i2c/slave/eeprom_slave.c

```diff
@@ -203,44 +205,44 @@ static int i2c_eeprom_slave_init(struct device *dev)
 	return 0;
 }
 
-#ifdef DT_INST_0_ATMEL_AT24
+#if DT_HAS_DRV_INST(0)
 
 static struct i2c_eeprom_slave_data i2c_eeprom_slave_0_dev_data;
 
-static u8_t i2c_eeprom_slave_0_buffer[(DT_INST_0_ATMEL_AT24_SIZE)];
+static u8_t i2c_eeprom_slave_0_buffer[(DT_INST_REG_SIZE(0))];
```

Comment:
Shouldn't these be converted to `DT_INST_PROP(0, size)`?

Reply:
Yep, good catch.  My script cant always tell the difference for a 'size' property.